cmake/DrogonUtilities.cmake

```diff
@@ -27,12 +27,19 @@ function(drogon_create_views arg)
                      ""
                      outputFile
                      ${f2})
+      if(ARGV3 STREQUAL TRUE)
+        set(ns "")
+      else()
+        set(ns -n ${ARGV3})
+        string(PREPEND outputFile "${ARGV3}_")
+      endif()
       add_custom_command(OUTPUT ${ARGV2}/${outputFile}.h ${ARGV2}/${outputFile}.cc
                          COMMAND drogon_ctl
                                  ARGS
                                  create
                                  view
                                  ${inFile}
+                                 ${ns}
```

Comment:
In my impression, the -n option and the --path-to-namespace option will not work at the same time, please ensure this by some tests.

Reply:
@tao-an 
I've just made following tests:
`
drogon_create_views(${PROJECT_NAME} ${CMAKE_CURRENT_SOURCE_DIR}/plugins/templates
                    ${CMAKE_CURRENT_BINARY_DIR}/plugins/templates test)
 `

The build will generate the files test_plugins_templates_test1.h and test_plugins_templates_test1.cpp with the class test::plugins::templates::test1. This class is also registered at DrClassMap.cc:::46

Following example is also ok:
`
HttpViewData data;
data.insert("message","TEST MESSAGE");
auto response = HttpResponse::newHttpViewResponse("test::plugins::templates::error", data);
callback(response);
`

Reply:
That's OK.

Reply:
@an-tao: I'm learning git. May you give me an hint, how to remove the chenge of Trantor?
Thanks

Reply:
```
cd trantor
git checkout master
git pull
cd ..
git add .
git commit -m "Update trantor"
git push
```